src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -522,17 +538,194 @@ UCS_PROFILE_FUNC_ALWAYS(ucs_status_t, uct_ib_mlx5_devx_reg_atomic_key,
                                          &memh->atomic_rkey);
 }
 
+void *uct_ib_mlx5_devx_mem_handle_thread_func(void *arg)
```

Comment:
why needed to move these functions to ib_mlx5dv_md.c?

Reply:
1. To make `ksm_data` an argument for `uct_ib_mlx5_devx_handle_mr_list_mt` since this structure contains `length`, `mr_num` and `mrs[]` arguments that are required in this function.
2. `ib_mlx5dv_md.c` is the only place where these functions are used, so I decided that this would be better place for them.

Reply:
1. Why is it needed to add ksm_data parameter to these functions to fix alignment?
2. We discussed it a while ago with @Artemy-Mellanox  and since these functions are generic we decided to keep them in ib_ms.c
3. Is that refactoring really needed to fix alignment issue?

Reply:
Moved these functions back to `ib_md.c`